src/selectedtests/datasource/mongo_wrapper.py

```diff
@@ -56,3 +56,19 @@ def task_mappings(self):
         :return: task_mappings collection.
         """
         return self.client.selected_tests.task_mappings
+
+    def test_mappings_project_config(self):
```

Comment:
I'm wondering whether this wrapper is really needed --> you aren't really abstracting away the database, but just hiding exactly which collections are being used, which seems to just make reasoning about things harder for the consumer. Just my 2 cents.

Reply:
I tend to agree with you on this. I was going off of what we do in dag-metrics: https://github.com/10gen/dag-metrics/blob/master/src/dagmetrics/datasource/mongo_wrapper.py. I'm not sure the `MongoWrapper` class adds much, but since we do it in other repos, I think it's something we should discuss as a team?